setup.sh

```diff
@@ -10,17 +10,19 @@ export LP_DIR=$ALIGN_HOME/lpsolve
 #export BOOST_LP=$ALIGN_HOME/boost
 export JSON=$ALIGN_HOME/json
 export GTEST_DIR=$ALIGN_HOME/googletest/googletest/
```

Comment:
We should probably leave this the way it was. You can have your own setup if you don;t want to use a VENV.

Reply:
I dont think it's a good idea if we duplicate the setups because of VENV, since they would likely diverge easily. I prefer to have one flexible codebase.